### PR TITLE
WIP: Add kiali yaml

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

And fix prometheus config

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: samples/addons/kiali.yaml [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+        sidecar.istio.io/inject: "false"
+      annotations:
+        checksum/config: b6f715c420df4b1ad451aff16d99961cff61f0016b250b5a4fa1d897252be282
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9090"
+        kiali.io/dashboards: go,kiali
+    spec:
+      serviceAccountName: kiali
+      containers:
+      - image: "quay.io/kiali/kiali:v2.1"
```
Comment (LiZhenCheng9527) [inline samples/addons/kiali.yaml:345]: Does kiali have a image of the latest tag?

**Comment by hzxuzhonghu** [comment]
@delavet

**Comment by delavet** [comment]
> @delavet to take a look at this issue

It seems that the current problem on the traffic graph is due to this `prometheus` job: 
https://github.com/kmesh-net/kmesh/blob/ab83d8ccffaca860a5a657d7873afddd5327a122/samples/addons/prometheus_recording_istio.yaml#L50

The `metric_relabel_configs` is only added in the `kubernetes-pods` job. Because we have `prometheus.io/scrape` annotation on the kmesh pod, this `kubernetes-pods` could record kmesh pods' metrics as expected. However, the `prometheus` job also record kmesh pods' metrics because these pods have `app: kmesh` label, but the `metric_relabel_configs` is not added to this job.

Does anyone know the necessity of this `prometheus` job?

File: samples/addons/prometheus_recording_istio.yaml [hunk 2]
```diff
@@ -299,7 +299,6 @@ data:
         source_labels:
         - __meta_kubernetes_pod_node_name
         target_label: node
-      - action: replace
       metric_relabel_configs:
```
Comment (hzxuzhonghu) [inline samples/addons/prometheus_recording_istio.yaml:302]: @delavet This is to replace app:kmeh to app:ztunnel? But i donot see we report metric with label `app:kmesh`

**Comment by hzxuzhonghu** [comment]
>job_name: prometheus 

It seems this is monitoring kubernetes pods

File: samples/addons/prometheus_recording_istio.yaml (hunk 2 above)
Comment (delavet) [reply samples/addons/prometheus_recording_istio.yaml:302]: https://github.com/kmesh-net/kmesh/blob/ab83d8ccffaca860a5a657d7873afddd5327a122/samples/addons/prometheus_recording_istio.yaml#L245

this job scrape metrics for pods with `prometheus.io/scrape` annotations (kmesh pods have this), and it has a `label_map` config to convert pod labels to metrics labels.

https://github.com/kmesh-net/kmesh/blob/ab83d8ccffaca860a5a657d7873afddd5327a122/samples/addons/prometheus_recording_istio.yaml#L50

this job scrape metrics for pods with `app: kmesh` labels (kmesh pods also have this), this does not have `label_map` config.

The result is this prometheus configuration result in scraping kmesh metrics twice, one with `app:kmesh` metrics label (and be relabeld to `app:ztunnel`), one does not.

**Pull request closed without merging**